RELEASE_NOTES.md

```diff
@@ -1,26 +1,36 @@
 # Cosmos SDK v0.47.0 Release Notes
 
-The main changes in this release are:
+✨ [**Official Release Announcenment**](https://blog.cosmos.network)
+
+💬 [**Release Discussion**](https://github.com/cosmos/community)
+
+## 📝 Changelog
+
+Check out the [changelog](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/CHANGELOG.md) for an exhaustive list of changes or [compare changes](https://github.com/cosmos/cosmos-sdk/compare/release/v0.46.x...release/v0.47.x) from last release.
```

Comment:
These links are going to be modified too? To point at the release instead of the release/v0.47.x branch?

Reply:
Updated here. I do not update the UPGRADING.md to a tag, as maybe we'll fine tune it.